docs/source/admin/upgrading.rst

```diff
@@ -27,51 +27,50 @@ Notify your users
 
 If you are using the default configuration where ``configurable-http-proxy``
 is managed by JupyterHub, your users will see service disruption during
-the upgrade process. You should notify them, and pick a time to do the
+the upgrade process. You will need notify them, and pick a time to do the
 upgrade where they will be least disrupted.
 
-If you are using a different proxy, or running ``configurable-http-proxy``
-independent of JupyterHub, your users will be able to continue using notebook
+If you are using a different proxy or running ``configurable-http-proxy``
+independent of JupyterHub, your users will be able to continue using a notebook
```

Comment:
"notebook servers" here is plural, I don't think it's proper to add "a" before it